components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/internal/DashboardMetadataProviderImpl.java

```diff
@@ -527,4 +527,25 @@ protected void unsetIdP(IdPClient client) {
         this.identityClient = null;
     }
 
+    public String getPdfHeader() {
+        try {
+            Object x = ((Map<String, Object>) configProvider.getConfigurationObject("wso2.status.dashboard"))
+                    .get("pdfHeader");
+            return x.toString();
+        } catch (ConfigurationException e) {
+            LOGGER.error("Error in getting the pdf header image", e);
```

Comment:
Why consume configurations of `wso2.status.dashboard`? 

- We shouldn't consume configurations from other namespaces, only from `wso2.dashboard` namespace.
- Don't declare dashboard configuration arbitrarily. Dashboard configurations from `deployment.yaml` is loaded into [DashboardConfigurations](https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/DashboardConfigurations.java) bean class. If you are introducing a new configuration, then that should be added into the `DashboardConfigurations` class.

Please fix this in the next PR.